lms/templates/problem.html

```diff
@@ -31,6 +31,10 @@ <h2 class="problem-header">
     % if answer_available:
     <button class="show"><span class='sr'>${_('Toggle Answer Visibility')}</span><span class="show-label">${_('Show Answer')}</span></button>
     % endif
+    <div class="reset-error">
+      <i class="fa fa-exclamation-triangle"></i>
+      <span class="reset-error">${_('You have to make an attempt before you reset the problem')}</span><span class="sr">${_('You have to make an attempt before you reset the problem')}</span>
+    </div>
```

Comment:
@mehrozezahid All the comments that i have given above for `problem_content.html` should be applied here.